test/functional/driver/better-sqlite3/better-sqlite3-native-binding.ts

```diff
@@ -0,0 +1,28 @@
+import 'reflect-metadata';
+import {closeTestingConnections, createTestingConnections, reloadTestingDatabases} from '../../../utils/test-utils';
+import {expect} from 'chai';
+import {join} from 'path';
+import {DataSource} from '../../../../src';
+
+const pathToBetterSqliteNode = join(__dirname, '../../../../../../node_modules/better-sqlite3/build/Release/better_sqlite3.node');
+
+describe.only('option nativeBinding for better-sqlite3', () => {
+
+    let connections: DataSource[];
+    before(async () => connections = await createTestingConnections({
+        entities: [],
+        enabledDrivers: ['better-sqlite3'],
+        driverSpecific: {
+            nativeBinding: pathToBetterSqliteNode,
+        },
+    }));
+    beforeEach(() => reloadTestingDatabases(connections));
+    after(() => closeTestingConnections(connections));
+
+    it('should use a the path set in nativeBindings to the node file', () => Promise.all(connections.map(async connection => {
+        expect((connection.options as any).nativeBinding).to.be.eql(pathToBetterSqliteNode);
+        // how to test if driver works?
```

Comment:
I don't know, maybe get the underlying better-sqlite driver and check if nativeBinding was set to that. It's not mandatory, you can skip it.